systemtests/run_test.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3   
+from subprocess import Popen, PIPE
+import time
+import os
+import signal
+from mcap_handler import McapHandler
+from datetime import datetime
+from plotter_class import Plotter
+from pathlib import Path
+
+
+if __name__ == "__main__":
+ 
+    path = Path(__file__)           #Path(__file__) should be "/home/github/actions-runner/_work/crazyswarm2/crazyswarm2/ros2_ws/src/crazyswarm2/systemtests/newsub.py" ; path.parents[0]=.../systemstests
+
+    bagname = 'bag_' + datetime.now().strftime('%d_%m_%Y-%H_%M_%S')
+    f8_bagname = "fig8_" + bagname
+    mt_bagname = "multitraj_" + bagname
+    src = "source " + str(path.parents[3].joinpath("install/setup.bash"))  # -> "source /home/github/actions-runner/_work/crazyswarm2/crazyswarm2/ros2_ws/install/setup.bash"
+    
+    #create the folder where we will record the different bags and the folder where the results pdf will be saved
+    bagfolder = (path.parents[3].joinpath(f"bagfiles/{bagname}"))  # /home/github/actions-runner/_work/crazyswarm2/crazyswarm2/ros2_ws/bagfiles/bag_d_m_Y-H_M_S
+    os.makedirs(bagfolder, exist_ok=True) 
+    os.makedirs(path.parents[3].joinpath("results"), exist_ok=True)  # /home/github/actions-runner/_work/crazyswarm2/crazyswarm2/ros2_ws/results
```

Comment:
delete old results and logs

Reply:
Alongside previous results pdfs and bagfiles, should the old logs from .ros/log also be deleted ?

Reply:
Yes, please. Ideally we would just have the results from the experiments that are being run.